cmd/k8s-operator/deploy/chart/templates/deployment.yaml

```diff
@@ -85,7 +82,7 @@ spec:
               value: {{ .Values.loginServer }}
             - name: OPERATOR_INGRESS_CLASS_NAME
               value: {{ .Values.ingressClass.name }}
-            {{- if .Values.oauth.clientSecret }}
+            {{- if or .Values.oauth.clientSecret .Values.oauthSecretVolume }}
```

Comment:
I also noticed that we were skipping past setting these `_FILE` env vars if we set `oauthSecretVolume` instead of `.oauth`. I am guessing we need this?